Symfony/CS/FixerInterface.php

```diff
@@ -75,6 +75,8 @@ public function getPriority();
     /**
      * Returns true if the file is supported by this fixer.
      *
+     * @param \SplFileInfo $file A \SplFileInfo instance
```

Comment:
Following recent Symfony's trend docs must be meaningful. Following PHPDoc the description of parameter is optional. Following Symfony if description gives us nothing it should be ommited.